apps/easypid/src/features/wallet/FunkeOpenIdPresentationNotificationScreen.tsx

```diff
@@ -150,6 +154,11 @@ export function FunkeOpenIdPresentationNotificationScreen() {
           return
         }
 
+        if (error instanceof PidIssuerPinLockedError) {
+          router.push('pinLocked')
```

Comment:
Need to be careful with `push` as you can go back. I think it should maybe be router.replace as there's no way back